packages/core/src/api/allnetwork/AllNetworkGetAddress.ts

```
@@ -263,64 +283,105 @@
       }
 
+      // call method
+      const response = await this.callMethod(methodName, params, param);
+
+      const result: AllNetworkAddress = {
+        ...response,
+        // @ts-expect-error
+        payload: {
+          ...response.payload,
+          ...dependOnMethodResults.reduce(
+            (acc, cur) => ({ ...get(cur, 'payload', {}), ...acc }),
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)